wp-content/themes/eighshift-task/src/blocks/custom/image-text/image-text-style.scss

```diff
@@ -0,0 +1,58 @@
+$block-image-text: (
+  vertical-spacing: 20px,
+);
+
+.block-image-text {
+  $this: &;
+
+  @include flex-container(
+    $layout: (12, 12),
+  );
+  @include media(tablet up) {
+    @include flex-container(
+      $layout: (5,6),
+      $vertical: center,
+    );
+  }
+
+
+  &__position--left {
+    #{$this}__visual {
+      order: 0;
+      margin-bottom: map-get-strict($block-image-text, vertical-spacing);
+
+      @include media(tablet up) {
+        margin-bottom: 0;
+      }
+    }
+
+    #{$this}__textual {
+      @include media(tablet up) {
+        margin-left: calc-grid-width(1);
+      }
+    }
+  }
+
+  &__position--right {
+    #{$this}__visual {
+      order: 1;
+    }
+
+    #{$this}__textual {
+      margin-bottom: map-get-strict($block-image-text, vertical-spacing);
+
+      @include media(tablet up) {
+        margin-bottom: 0;
+        margin-right: calc-grid-width(1);
```

Comment:
if you put this as a padding, whan you dont need to fiddle with grid columns. just put columns equal size 6 and 6. Simpler to maintain

Reply:
But then I lose the required spacing, because with padding property I'm overriding the columns gutter padding.

Reply:
👍